frontend/src/components/ProductCard.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import {
+  Card,
+  CardContent,
+  Typography,
+  Button,
+  CardMedia,
+} from "@mui/material";
+
+interface ProductCardProps {
+  name: string;
+  price: number;
+  country_of_origin: string;
+  description: string;
+  fruit_or_vegetable: string;
+  imageId: string;
+  expiry_date: string;
+  onAddToCart: () => void;
+}
+
+const ProductCard: React.FC<ProductCardProps> = ({
+  name,
+  price,
+  country_of_origin,
+  description,
+  fruit_or_vegetable,
+  imageId,
+  expiry_date,
+  onAddToCart,
+}) => {
+  return (
+    <Card>
+      <CardMedia
+        component="img"
+        height="140"
+        image={`http://localhost:8000/api/images/${imageId}`}
+        alt={name}
```

Comment:
please introduce an env property or backend url and use it in whole code http://localhost:8000/api/images